### PR TITLE
Shenli/fix beego

### PR DESCRIPTION
**Pull request by shenli**
1. Support alias table name in update multiple table
2. Support LIKE BINARY

### CODE REVIEW

File: expression/expressions/like.go [hunk 1]
```diff
@@ -103,7 +103,12 @@ func (p *PatternLike) Eval(ctx context.Context, args map[interface{}]interface{}
 		}
 		spattern, ok := pattern.(string)
 		if !ok {
```
Comment (siddontang) [inline expression/expressions/like.go]: you can use switch type 
switch x := pattern.(type) 
 case string:
case []byte:
default:

File: stmt/stmts/update.go [hunk 2]
```diff
@@ -279,7 +296,7 @@ func (s *UpdateStmt) Exec(ctx context.Context) (_ rset.Recordset, err error) {
 			end := start + len(tbl.Cols())
 			data := rowData[start:end]
 			start = end
-			tcols, err2 := getUpdateColumns(tbl, s.List, isMultipleTable)
+			tcols, err2 := getUpdateColumns(tbl, s.List, isMultipleTable, tblAliasMap)
```
Comment (siddontang) [inline stmt/stmts/update.go]: why not use `GetFieldIndex` or others in fields package.

I thin these can do the same thing, @qiuyesuifeng, how do you think?

Comment (qiuyesuifeng) [inline stmt/stmts/update.go]: Yes, but we have to do some update in field package to support these,
so i think it is better in a new PR.

Comment (siddontang) [inline stmt/stmts/update.go]: maybe we can add a TODO here. @shenli

Comment (shenli) [inline stmt/stmts/update.go]: ok

**Comment by shenli** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**